OpenSim/Common/IO.cpp

```diff
@@ -730,6 +730,21 @@ IO::CwdChanger::CwdChanger(IO::CwdChanger&& tmp) :
     std::swap(this->_existingDir, tmp._existingDir);
 }
 
+IO::CwdChanger& IO::CwdChanger::operator=(CwdChanger&& tmp) {
+    this->_existingDir.clear();
+    std::swap(this->_existingDir, tmp._existingDir);
+    return *this;
+}
+
```

Comment:
My understanding of the use case of this class is that we'll bake the change/restore in the construction/destruction, why do we need move and reset is a bit surprising but may become clear from the rest of the changes.  Could we do without these?